cherab/generomak/machine/plasma.py

```diff
@@ -0,0 +1,56 @@
+import os
```

Comment:
Can you add the license header to this file please? Take it from any of the other `.py` files, but please make sure the dates are current.

Reply:
Added licence which I coppied from a recently added file